### DISCUSSION TITLE
MySQL 9.0

### DISCUSSION THREAD

**Discussion by dapzz-id** [leokhoa/laragon#871] [General]
Bagaimana cara menghubungkan MySQL versi 9.0 atau versi diatas 8.0?

**Comment by MeoNguOfficial** [comment]
Just update it
but i still have same question

**Comment by borhanuddin** [comment]
Saya telah buat manual untuk masalah ini boleh rujuk URL: https://github.com/leokhoa/laragon/discussions/932#discussion-7699055
